markdown/extensions/tables.py

```diff
@@ -85,7 +85,38 @@ def _split_row(self, row, border):
                 row = row[1:]
             if row.endswith('|'):
                 row = row[:-1]
-        return row.split('|')
+        return self._split(row, '|')
+
+    def _split(self, row, marker):
+        """ split a row of text with some code into a list of cells. """
+        if self._row_has_unpaired_backticks(row):
+            # fallback on old behaviour
+            return row.split(marker)
+        escaped = False
+        elements = []
+        current = ''
+        for i in row:
+            if i == '`' and (current == '' or current[-1] != '\\'):
+                # Found an un-escaped backtick
+                escaped = not escaped
+            if i == marker and not escaped:
+                elements.append(current)
+                current = ''
+            else: 
```

Comment:
The [Flake8 test](https://travis-ci.org/waylan/Python-Markdown/jobs/86851063) is indicating this has trailing whitespace.